### PR TITLE
Use value parameter for unjailing nodes

### PR DESCRIPTION
**Pull request by popenta**
Now setting value on transaction when unjailing nodes.

### CODE REVIEW

File: multiversx_sdk_cli/dependencies/modules.py [hunk 1]
```diff
@@ -241,7 +241,7 @@ def install(self, overwrite: bool) -> None:
         self._install_sc_meta()
         self._install_wasm_opt()
         self._install_twiggy()
-        self._install_sc_meta_deps()
+        show_message("Please also install all the necessary dependencies required for `sc-meta` to run properly bu running the following command: `sc-meta install all`.")
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/dependencies/modules.py:244]: Typo.

File: multiversx_sdk_cli/delegation/staking_provider.py [hunk 2]
```diff
@@ -171,11 +171,13 @@ def prepare_transaction_for_unjailing_nodes(self, owner: IAccount, args: Any) ->
...
 
         public_keys = self._load_validators_public_keys(args)
+        amount = int(args.value)
 
         tx = self._factory.create_transaction_for_unjailing_nodes(
             sender=owner.address,
             delegation_contract=delegation_contract,
-            public_keys=public_keys
+            public_keys=public_keys,
+            amount=amount
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/delegation/staking_provider.py:180]: Should have been a separate PR from the one that alters `sc-meta` installation flow.

File: multiversx_sdk_cli/dependencies/modules.py (hunk 1 above)
Comment (andreibancioiu) [inline multiversx_sdk_cli/dependencies/modules.py:244]: Should have been a separate PR from the one adjusting the `amount` for unjailing (for changelog / release note purposes).

File: multiversx_sdk_cli/delegation/staking_provider.py (hunk 2 above)
Comment (popenta) [reply multiversx_sdk_cli/delegation/staking_provider.py:180]: I'll move the alteration of sc-meta in another PR.

File: multiversx_sdk_cli/dependencies/modules.py (hunk 1 above)
Comment (popenta) [reply multiversx_sdk_cli/dependencies/modules.py]: fixed

**Review by andreibancioiu — APPROVED** [review]

**Review by danidrasovean — APPROVED** [review]

**Pull request merged**